upf/upf.h

```diff
@@ -769,6 +775,9 @@ typedef struct
   upf_upip_res_t *upip_res;
   mhash_t upip_res_index;
 
+  /* teid availability table */
+  u32 *teid_table;
+
```

Comment:
there should be no need for this, use the v4/v6_tunnel_by_key table instead.

TEIDs are always bound to IPs. A TEID table therefore need to take the IP in consideration as well.

beware of the the special case of F_TEID with CH+CHID+V4+V6 bits set